pkg/client/dtclient/settings_client.go

```diff
@@ -223,6 +238,14 @@ type (
 		TotalCount int        `json:"totalCount"`
 	}
 
+	Accessor struct {
+		Type TypeAccessor `json:"type"`
+	}
+	PermissionObject struct {
+		Permissions []TypePermissions `json:"permissions"`
+		Accessor    *Accessor         `json:"accessor,omitempty"`
```

Comment:
Can the accessor actually be undefined? I would expect that it is always set to "all-users", "group" or "user".
When looking at the POST API, a missing accessor leads to a contraint violation

Reply:
In a PUT it can be undefined. Since you would send the request at for example the all-users endpoint.

Reply:
Maybe a follow up if we should add different structs for this case.

Reply:
I agree, but also keep in mind monaco should not reimplement domain constrains of the payload.